tests/queries/0_stateless/02345_implicit_transaction.sql

```diff
@@ -3,7 +3,7 @@
 CREATE TABLE landing (n Int64) engine=MergeTree order by n;
 CREATE TABLE target  (n Int64) engine=MergeTree order by n;
 CREATE MATERIALIZED VIEW landing_to_target TO target AS
-    SELECT n + throwIf(n == 3333)
+    SELECT n + throwIf(n == 3333) AS n
```

Comment:
This is also concerning. Why are nonrefreshable views affected by the column's aliasing changes?

Reply:
Added compatibility setting in https://github.com/ClickHouse/ClickHouse/pull/58934 .